VizualizingData/3_different_plot_types.Rmd

```diff
@@ -0,0 +1,132 @@
+---
+title: ""
+output:
+  html_document:
+    highlight: tango
+    toc: true
+    toc_float: true
+    css: ../css/style-chapters.css
+--- 
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(ggplot2)
+```
+
+# Plot types
+
+Choosing the right type of plot, depends on your data. Wheather you have continous data, a factor, grouped variables, will define what type of plots you can do. Here, we present a number of common plot types. You can find the code to plot these figures below.
+Here is a guide that can help you choose the plot type (if we still want to keep it...)
+
+In ggplot, the `geom_xxx()` funciton defines the type of plot that is drawn. in the example below, we use `geom_point()`, which makes dots.
+
+```{r simple plot2, eval=TRUE, echo=FALSE}
+ggplot(data = iris, aes(x = Sepal.Length, y = Petal.Length, colour = Species, shape = Species)) +
```

Comment:
Recently, there has been a push to stop using the iris dataset. The problem with it is that Fisher was a racist and this is the citation: Fisher, R. A. (1936) The use of multiple measurements in taxonomic problems. **Annals of Eugenics**, 7, Part II, 179–188. If only Anderson had published the data rather than leaving it to Fisher. A great alternative is the penguin dataset - see https://github.com/allisonhorst/palmerpenguins also on CRAN

Some of my examples need updating too

Reply:
Yes, I'll use another data set. I have not worked on this for a while now.